tests/index_test.py

```diff
@@ -101,13 +109,7 @@ def test_multi_lookup(self):
 
     def test_save(self):
         locale_count = ak.get_config()["numLocales"]
-        d = f"{os.getcwd()}/save_index_test"
-        # make directory to save to so pandas read works
-        os.mkdir(d)
-
-        idx = ak.Index(ak.arange(5))
-        idx.save(f"{d}/idx_file.h5")
-        self.assertEqual(len(glob.glob(f"{d}/idx_file_*.h5")), locale_count)
-
-        # clean up test files
-        rmtree(d)
+        with tempfile.TemporaryDirectory(dir=IndexTest.ind_test_base_tmp) as tmp_dirname:
+            idx = ak.Index(ak.arange(5))
+            idx.to_hdf(f"{tmp_dirname}/idx_file.h5")
```

Comment:
Should we add a `to_parquet` version of this test as well?

Reply:
Again that functionality is handled by the parquet testing file.